### ISSUE TITLE
Plugin is getting access issue

### ISSUE THREAD

**Issue by Tooldad** [danieleborgo/OctoPrint-ShutdownButtonLEDBuzzer#5]
Thank you for putting this plugin together.  Unfortunately I am having an issue with it and hoping you can help.  I've tried uninstalling it and reinstalling it but that didn't help.  

I'm thinking I am missing some sort of access, I tried this command, but it didn't help...
sudo usermod -a -G gpio $USER

Note: When I connect to my Octopi with Putty, I have to use "pi" as the userid.  But I notice I have a different ID when I look at my main screen in Octoprint (top right corner).  I think that is normal, but just checking.

Both of the items at the bottom of the ShutdownButtonLEDBuzzer screen (I2C and SPI) say "disabled".

Here is the log.....
2022-01-18 14:41:23,616 - octoprint.plugins.dashboard - INFO - Dashboard started
2022-01-18 14:41:23,634 - octoprint.JsFrontendErrors(Dash) - INFO - Js Logger (Dash) started
2022-01-18 14:41:23,784 - octoprint.plugin - ERROR - Error while calling plugin shutdownbuttonledbuzzer
Traceback (most recent call last):
  File "/home/pi/oprint/lib/python3.7/site-packages/octoprint/plugin/__init__.py", line 271, in call_plugin
    result = getattr(plugin, method)(*args, **kwargs)
  File "/home/pi/oprint/lib/python3.7/site-packages/octoprint/util/__init__.py", line 1737, in wrapper
    return f(*args, **kwargs)
  File "/home/pi/oprint/lib/python3.7/site-packages/octoprint_shutdownbuttonledbuzzer/__init__.py", line 81, in on_after_startup
    self.__setup()
  File "/home/pi/oprint/lib/python3.7/site-packages/octoprint_shutdownbuttonledbuzzer/__init__.py", line 108, in __setup
    self.__close_button()
  File "/home/pi/oprint/lib/python3.7/site-packages/octoprint_shutdownbuttonledbuzzer/__init__.py", line 160, in __close_button
    GPIO.remove_event_detect(self.__button_pin)
RuntimeError: No access to /dev/mem.  Try running as root!
2022-01-18 14:41:23,784 - octoprint.plugins.themeify - INFO - Themeify initialized.

**Comment by Tooldad** [comment]
I just tried a completely new install of octopi/octoprint.  Then only installed this plugin.  I am getting the same error.  Here is my log...

2022-01-18 11:30:34,150 - octoprint.plugins.pluginmanager - INFO - Loaded notice data from disk, was still valid
2022-01-18 11:30:34,175 - octoprint.plugins.softwareupdate - INFO - Saved version cache to disk
2022-01-18 11:30:34,178 - octoprint.plugin - ERROR - Error while calling plugin shutdownbuttonledbuzzer
Traceback (most recent call last):
  File "/home/pi/oprint/lib/python3.7/site-packages/octoprint/plugin/__init__.py", line 271, in call_plugin
    result = getattr(plugin, method)(*args, **kwargs)
  File "/home/pi/oprint/lib/python3.7/site-packages/octoprint/util/__init__.py", line 1737, in wrapper
    return f(*args, **kwargs)
  File "/home/pi/oprint/lib/python3.7/site-packages/octoprint_shutdownbuttonledbuzzer/__init__.py", line 81, in on_after_startup
    self.__setup()
  File "/home/pi/oprint/lib/python3.7/site-packages/octoprint_shutdownbuttonledbuzzer/__init__.py", line 108, in __setup
    self.__close_button()
  File "/home/pi/oprint/lib/python3.7/site-packages/octoprint_shutdownbuttonledbuzzer/__init__.py", line 160, in __close_button
    GPIO.remove_event_detect(self.__button_pin)
RuntimeError: No access to /dev/mem.  Try running as root!
2022-01-18 11:30:34,238 - octoprint.server.preemptive_cache - INFO - Preemptively caching / (ui _default) for {'base_url': 'http://octopi2.local/', 'path': '/', 'query_string': 'l10n=en'}
2022-01-18 11:30:40,028 - octoprint.server.preemptive_cache - INFO - ... done in 5.79s
2022-01-18 11:31:44,825 - octoprint.server.util.flask - INFO - Passively logging in user tooldad2 from ::ffff:192.168.86.21
2022-01-18 11:31:44,826 - octoprint.access.users - INFO - Logged in user: tooldad2
2022-01-18 11:31:45,475 - octoprint.server.util.sockjs - INFO - New connection from client: ::ffff:192.168.86.21
2022-01-18 11:31:45,536 - octoprint.server.util.flask - INFO - Passively logging in user tooldad2 from ::ffff:192.168.86.21
2022-01-18 11:31:45,537 - octoprint.access.users - INFO - Logged in user: tooldad2
2022-01-18 11:31:46,137 - octoprint.server.util.sockjs - INFO - User tooldad2 logged in on the socket from client ::ffff:192.168.86.21
2022-01-18 11:31:46,679 - octoprint.plugins.shutdownbuttonledbuzzer - INFO - Refresh request received

**Comment by danieleborgo** [comment]
Hi,
the problem was mine, not yours, so the installation you did is perfectly ok. That error is, differently by what it tells, related to the fact I tried to deallocate the button before creating it (source [here](https://stackoverflow.com/questions/62959885/python-error-runtimeerror-no-access-to-dev-mem-try-running-as-root-tried-a)).

I fully moved the plugin to _gpiozero_ and I just uploaded the fix. I suggest you to fully uninstall it and install it again.

Thank you for the bug report.
